main.sh

```diff
@@ -169,9 +156,13 @@ run_IOR(){
         for i in "${IOR_PROC_PER_CLIENT[@]}"; do
             no_of_ps=$(($DAOS_CLIENTS * $i))
 	    echo
-            cmd="orterun --timeout 600 $VERBS_CLIENT_PARAM 
-            -np $no_of_ps --map-by node  --hostfile Log/$SLURM_JOB_ID/daos_client_hostlist ior -a DAOS -b $BL_SIZE  -w -r -i 3 -o daos:testFile 
-            -t $size --daos.cont $(uuidgen) --daos.destroy --daos.group daos_server --daos.pool $POOL_UUID  --daos.svcl $POOL_SVC -vv"
+            cmd="mpirun -np $no_of_ps -map-by node 
+                 -hostfile Log/$SLURM_JOB_ID/daos_client_hostlist
+                 ior
```

Comment:
How are we building 'ior/mdtest' now and where are we planning to keep the ior/mdtest binaries?
As we are using mpich, I would suggest to keep and build the mpich and ior binaries at same location and use the same node to avoid incompatibility of any sort.

Reply:
The ior/mdtest are built and installed in the daos/install dir and I think that is appropriate since they are daos dependent.

Reply:
I would suggest as we are using mpich in this case and ior needs to be built using that same mpich, they both kind of becomes the dependencies. And in that case, either have both mpich and ior binaries under 'daos/install' or have both of them under '~/utils/' on frontera.
But just a suggestion. I feel it will avoid future confusions of mismatched versions.

Reply:
yep, we are building mpich and ior/mdtest on frontera now to avoid incompatibility.